### PR TITLE
Portuguese Translation

### PR DESCRIPTION
**Pull request by luismilanooliveira**
This is the portuguese version of the guide after some work I did. I'm pretty sure there is room to improvement, but most of the work is done!
I do not know if there is more portuguese speakers around here, but some help in reviewing is very welcome!

I tried to follow the style of the preceding translations (german & french) in regard to keeping intact some english words. There are programming-specific terms I think there is no point in translating. Specifically, I kept all haskell-specific vocabulary (functors, applicative, monads, typeclasses, etc), as the person learning Haskell will be seeing them on a "everyminute" basis. 
I tried to add some emphasis(i.e. put in italic) in words in english pertaining to the text properly speaking.

Maybe it would be better to wait some time in the case someone has something to say before doing the merge.

### CODE REVIEW

File: guide-pt.md [hunk 1]
```diff
@@ -0,0 +1,812 @@
...
+
+## Diretrizes para a Comunidade
+
+[Leia a postagem de Chris Done sobre ensino](http://chrisdone.com/posts/teaching) (Em inglês)
+
+Seja gentil e cortês. Ser cruel ou rude assusta as pessoas e as desencoraja a participar.
+
+Criticismo barato beneficia apenas a pessoa que o emite, não a pessoa que o recebe.
+
+Não descreva coisas como "fáceis" ou "triviais". Você fará pessoas se sentirem mal por ter que trabalhar duro por progredir naquilo. Pessoas que aprendem lentamente frequentemente são aquelas que aprendem melhor, isso é algo para ser celebrado!
```
Comment (fenak) [inline guide-pt.md]: I believe this could be better put this way:

Você fará pessoas se sentirem mal por terem que trabalhar duro para progredirem.

**Comment by fenak** [comment]
Looks great! I made a comment on a line (maybe a small improvement?) but overall is very good!

:+1:

**Comment by luismilanooliveira** [comment]
Thanks, @fenak! That part was very poorly phrased. =p

**Comment by fenak** [comment]
Yeah, that is way better!

:smile:

**Comment by bitemyapp** [comment]
Thank you @luismilanooliveira and @fenak ! :tada: :+1: :v:

**Pull request merged**